doc/nrf/security/tfm.rst

```diff
@@ -87,13 +87,18 @@ Logging
 *******
 
 TF-M employs two UART interfaces for logging: one for the secure part (MCUboot and TF-M), and one for the non-secure application.
-The logs arrive on different COM ports on the host PC.
+By default, the logs arrive on different COM ports on the host PC.
+See the :ref:`ug_tfm_manual_VCOM_connection` for more details.
 
+Alternatively, you can configure the TF-M to connect to the same UART as the application by using the :kconfig:option:`CONFIG_TFM_SECURE_UART_SHARE_INSTANCE` Kconfig option.
```

Comment:
Not sure exactly how to phrase this. Because the default uart instance used by the application is instance 0, and by default TF-M use uart instance 1.
It's possible to change the TF-M uart instance to instance 0 with CONFIG_TFM_SECURE_UART0.
When the application and TF-M uart instance is the same then TFM_SECURE_UART_SHARE_INSTANCE should be selected, this is done automatically by the config system.
This will cause TF-M to only output logging during boot, and fatal exceptions.

Reply:
So documentation does not need to mention TFM_SECURE_UART_SHARE_INSTANCE, but it could mention CONFIG_TFM_SECURE_UARTX so users know about that option?